PixaView/Controller/PopularSortButtonController.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  PopularSortButton.swift
+//  PixaView
+//
+//  Created by Дмитрий Игнатьев on 05.12.2021.
+//
+
+import UIKit
+import TestFramework
+
+protocol SortImageProtocol: AnyObject{
```

Comment:
I believe you can rename it to SortViewControllerDelegate

Reply:
done